build/changelog/ChangelogBuilder.php

```diff
@@ -67,9 +67,7 @@ private function cleanJSON($arr)
             if ($x->type == 'NEW_SERVICE') {
```

Comment:
This `for` loop isn't really doing anything anymore. You could just set `$this->newServiceFlag` to `count(array_filter($arr, function($change) { return $change->type === 'NEW_SERVICE'; }) > 0`  and return the array that was passed in.